doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst

```diff
@@ -0,0 +1,67 @@
+Pythagorean Forest
+==================
+
+.. figure:: icons/pythagorean-forest.png
+
+Pythagorean forest for visualising random forests.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Random Forest**
+
+Classification / regression tree models as random forest.
```

Comment:
It should be indented. It's easy to see the fault if you process it into HTML, or just [preview the result](https://github.com/biolab/orange3/pull/1511/files?short_path=d23dea2#diff-d23dea23dcda62bd206bfa2dd75c66bf) (locate this file once the page loads) here on Github. The description falls out of the list (and a new list is begun if more than one signal). This is clearly unprofessional, semantically incorrect, and it looks the weirder, depending upon the stylesheet.

This is how it should look like: https://github.com/biolab/orange3/blob/3ba5ab2e032440d516dbb916caf50a47194d94de/doc/widgets/classify/classificationtree.md

This is how it looks now: https://github.com/biolab/orange3/blob/master/doc/visual-programming/source/widgets/classify/classificationtree.rst

Let me know if you can spot the difference.